derex/discovery/config.py

```diff
@@ -96,7 +96,7 @@ def local_compose_options(
             return {
                 "options": options,
                 "name": "discovery",
-                "priority": "<local-derex",
+                "priority": "_end",
```

Comment:
>I think this should be <local-project or >base-project otherwise we can't be sure that a local docker-compose.yml file will have any effect.
>The fact that it breaks if no local docker-compose.yml exists is a derex.runner issue (Abstract-Tech/derex.runner#131).
> Also the variant key is useless and redundant.

Reply:
Good catch! Fixed